e2e/helm.e2e.spec.ts

```diff
@@ -0,0 +1,100 @@
+import path from 'path';
+import {
+  ElectronApplication, BrowserContext, _electron, Page, Locator
+} from 'playwright';
+import { test, expect } from '@playwright/test';
+import { createDefaultSettings, kubectl, helm } from './utils/TestUtils';
+
+let page: Page;
+const defaultReportFolder = path.join(__dirname, 'reports/');
```

Comment:
Would it make sense to just write reports into `/tmp/cirrus-ci-build/e2e/reports/` directly?
(Alternatively, `ln -s /tmp/cirrus-ci-build/e2e/reports/ /home/ranchertest/e2e/reports/` before running the tests…)

Reply:
It's nice to have the trace package per test spec.
The issue on Cirrus CI, the artifact uploader cannot access folders outside of the main runner path (`/tmp/cirrus-ci-build/*`). It seems related to the secutiry stuff... 
Your suggestion make sense, instead copying/moving files around, just create a symlink should do the trick... I'll give a try.